samza-core/src/main/java/org/apache/samza/config/ShellCommandConfig.java

```diff
@@ -58,16 +58,14 @@
   public static final String ENV_EXECUTION_ENV_CONTAINER_ID = "EXECUTION_ENV_CONTAINER_ID";
 
   /**
-   * Set to "true" if cluster-based job coordinator dependency isolation is enabled. Otherwise, will be considered
-   * false.
+   * Set to "true" if split deployment feature is enabled. Otherwise, will be considered false.
    *
-   * The launch process for the cluster-based job coordinator depends on the value of this, since it needs to be known
-   * if the cluster-based job coordinator should be launched in an isolated mode. This needs to be an environment
-   * variable, because the value needs to be known before the full configs can be read from the metadata store (full
-   * configs are only read after launch is complete).
+   * The launch process for the cluster-based job coordinator and job container depends on the value of this, since it
+   * needs to be known if the cluster-based job coordinator and job container should be launched in an isolated mode.
```

Comment:
Instead of "isolated", could you please use the term "split deployment". I think the term isolation will generally be replaced by the term "split deployment", so it might get confusing what "isolation" refers to in the future.

Reply:
sure, thanks for pointing out.

Reply:
The related terms are fixed.